keepercore/MANIFEST.in

```diff
@@ -1,2 +1 @@
-include requirements-dev.txt
-include requirements.txt
```

Comment:
I remember that I had to add those files manually here. Can't remember why...

Reply:
Ah right, I didn't see that there was an existing MANIFEST and just replaced it with the static content. Those two includes are required for the source distribution to be able to include the txt files in `setup.py`.